README.md

```diff
@@ -175,8 +179,29 @@ Check out the [sample app](./test-app) to see a webpack config in action or read
 - [tslint](./packages/tslint)
 - [typescript](./packages/typescript)
 - [uglify](./packages/uglify)
-- [webpack](./packages/webpack)
 
+## [Helpers](./packages/webpack#helpers).
```

Comment:
Is there a reason for the trailing period? (See also shorthand setters 😉)

Reply:
Nope ;-)